addon/index.js

```diff
@@ -157,6 +158,15 @@ function initServerEnvironmentPreference() {
   });
 }
 
+function openOnboardingTab() {
+  // TODO: Settings is populated async, so we need a global afterSettingsReady
+  // promise that this function can wait on.
+  tabs.open({
+    url: SERVER_ENVIRONMENTS.production.BASE_URL + '/onboarding',
```

Comment:
this seems like a good candidate for a followup bug, actually. getting pretty late on friday.